types/draco3d/index.d.ts

```diff
@@ -105,8 +114,8 @@ export interface Decoder {
     GetAttributeUInt16ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
     GetAttributeUInt32ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
     GetEncodedGeometryType: (buffer: DecoderBuffer) => GeometryType;
-    GetAttributeId: (mesh: Mesh, attributeType: number) => number;
-    GetAttribute: (mesh: Mesh, id: number) => Attribute;
+    GetAttributeId: (mesh: PointCloud, attributeType: number) => number;
+    GetAttribute: (mesh: PointCloud, id: number) => Attribute;
```

Comment:
See other comment — probably want `Mesh | PointCloud` here.

Reply:
This is based on the following
https://github.com/google/draco/blob/master/src/draco/javascript/emscripten/decoder_webidl_wrapper.h#L130
https://github.com/google/draco/blob/master/src/draco/javascript/emscripten/decoder_webidl_wrapper.cc#L125